events-processor/main.go

```diff
@@ -48,10 +55,24 @@ func main() {
 
 	defer sentry.Flush(2 * time.Second)
 
+	memCache, err = cache.NewCache(cache.CacheConfig{
+		Context: ctx,
+		Logger:  logger,
+	})
+	if err != nil {
+		logger.Error("Error creating the cache", slog.String("error", err.Error()))
+		utils.CaptureError(err)
+		panic(err.Error())
```

Comment:
You can now do this:
```suggestion
        utils.LogAndPanic(logger, err, "Error creating the cache")
```